components/encoder/incremental/incremental_encoder.go

```diff
@@ -208,11 +211,12 @@ func (e *Encoder) Start(ctx context.Context) {
 	// 0 -> same state
 	// x -> impossible state
 
-	chanA := make(chan board.Tick)
-	chanB := make(chan board.Tick)
-
-	e.A.AddCallback(chanA)
-	e.B.AddCallback(chanB)
+	ch := make(chan board.Tick)
+	err := e.board.StreamTicks(e.cancelCtx, e.interrupts, ch, nil)
```

Comment:
Part of me wants to put a `/* extra */` in with the `nil` to remind me what it is, and part of me says "once I'm used to `StreamTicks` I won't need that comment any more, so leave it out." I can't tell which approach is right, just something to think about.